packages/gatsby/src/components/navigation.js

```diff
@@ -83,9 +83,7 @@ const MenuEntry = styled(Link)`
 `;
 
 const Content = styled.div`
-  & > * {
-    overflow: auto;
```

Comment:
It's not clear what this was for. I didn't notice anything wrong.

It's required in order to get the `position: sticky` to work on the ToC <aside> element.